frontend/models/product-list/index.ts

```diff
@@ -174,7 +174,7 @@ export function getProductListPath(
 ): string {
    switch (productList.type) {
       case Enum_Productlist_Type.Tools: {
-         return `/Store/Tools/${productList.handle}`;
+         return `/Tools/${productList.handle}`;
       }
       case Enum_Productlist_Type.Marketing: {
          return `/Store/${productList.handle}`;
```

Comment:
I think this is the only difference between this pull and #325

https://github.com/iFixit/react-commerce/pull/325/files#diff-3cc5a6bae5107749a587f5a63495efd258b9c9b13c80141b605d0e996d5ab0b8

Reply:
Not true, @andyg0808 pointed out another class of differences [here](https://github.com/iFixit/react-commerce/pull/325#pullrequestreview-998937888)